control-plane/api/common/configentry_webhook_test.go

```diff
@@ -115,7 +115,7 @@ func TestValidateConfigEntry(t *testing.T) {
 					},
 				},
 			},
-				logrtest.TestLogger{T: t},
+				logrtest.NewTestLogger(t),
```

Comment:
An update to this library removed this direct access in favor of `NewTestLogger`. The behavior is unchanged.